gaj23/face-it
# Ch/feature/footer

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

Sets up initial layout and foundation for the app, with React, Router, and Cypress.  The App component was modified to render with Router and links to categories are clickable but do not have data yet displayed.  The title and icon show with a navigation bar at the top.  The three categories are shown and are responsive according to pixel width of the window.  

### Problems Encountered & Resulting Solutions  
> State any problems encountered 

The server for the Makeup API we are using was causing issues before we realized that it is set up to reach a limit of calls that we were exceeding. We proceeded to download the data to post on a backup server (in another repo).

> Describe your code changes in detail for reviewers.

File folders within the Components folder of the project are updated with Component functionality we anticipate adding in future iterations.  The static comp of the main display is complete in the App component.  The icon component "Logo" is an addition as well.  

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

All code is new aside from the starter code from the Create React app.

### How should this be tested?
> What steps need to be taken to confirm functionality? 

In your terminal, enter `npm start` once you are in the project directory to open the server.  You should see the static page in your browser (localhost: 3000).

### Steps To Take
> Any questions and possible steps moving forward.



#### Checklist:
